com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/app/discovery/ZigBeeNodeServiceDiscoverer.java

```diff
@@ -515,6 +514,9 @@ private boolean requestNeighborTable() throws InterruptedException, ExecutionExc
      * @throws InterruptedException
      */
     private boolean requestRoutingTable() throws InterruptedException, ExecutionException {
+        if (!supportsManagementRouting) {
```

Comment:
Why is this necessary? If the task is removed due to the command not being supported, then it should not be necessary to check this again.